src/pages/Swap.tsx

```diff
@@ -604,7 +612,7 @@ function Swap(): ReactElement {
             ? "0"
             : formatUnits(
                 formState.to.value,
-                tokensMap[formState.to.symbol]?.decimals,
+                tokenSymbolToTokenMap[formState.to.symbol]?.decimals,
```

Comment:
With the check on L524, the optional here could go.

Reply:
This was one occurrence I couldn't figure out how to remove the optional check. when testing the app would break. This is one that needs to be revisited in the future.

Reply:
You're right. This page needs to be broken up anyway.